stories/ag/ListingCell.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { ListingCell } from 'react-ui-ag/src'
+import { action } from '@storybook/addon-actions'
+import Star from './StarRatingItem'
+
+export const DefaultListingCell = (
+  <ListingCell
+    onCardClick={action('card click action')}
+    onFavoriteClick={action('on favorite click')}
+    listingDetails={{
+      listingId: 1,
+      rating: 4,
+      numRatings: 10,
+      price: '$2000+',
+      title: 'Cool Apartment',
+      bedroomText: '3 bedrooms',
+    }}
+    RatingItem={Star}
+    viewType="map"
+    ctaSection={[
+      {
+        children: '404-378-1428',
+        onClick: action('Phone CTA action'),
+        type: 'phone',
```

Comment:
Is the type property of ctaSection ever being used? I don't see it being accessed anywhere else.

Reply:
yeah we use it for the `data-tid`